addons/packages/ako-operator/1.4.0/README.md

```diff
@@ -42,3 +129,81 @@ The following configuration values can be set to customize the ako-operator inst
 ## Usage Example
 
 This walkthrough guides you through using ako-operator...
+
+### Verify your configurations
+
+After the package is installed and reconcile successfully, two AKODeploymentConfigs are deployed globally.  
+You can list and inspect them with `kubectl get akodeploymentconfig`. They are the configuration interfaces for the ako-operator.
+In general,
+
+* `install-ako-for-management-cluster` specifies the parameters to deploy AKO statefulset in the management cluster, which
+  then provides L4/L7 load balancing for applications.
+* `install-ako-for-all` specifies the parameters to deploy AKO statefulsets and reconcile users in each future workload clusters. The ako-operator
+  passes these parameters to each workload clusters via addon secrets.
+
+In the management cluster, ensure that there is no error level message in the ako-operator manage log.
```

Comment:
Instead of checking this one, users should check if AKO statefulset got deployed in cluster successfully and check AKO's log.